src/components/form/form.component.js

```diff
@@ -12,9 +12,15 @@ import { withValidations } from '../validations';
 import ElementResize from '../../utils/helpers/element-resize/element-resize';
 import StyledForm, { StyledFormFooter, StyledAdditionalFormAction } from './form.style';
 
-const FormContext = React.createContext();
```

Comment:
I guess there's a reason why we need to revert to React's legacy context API here?

Reply:
Yeah sorry I mentioned it in a message offline, it breaks a lot of the I had to revert some of the legacy context functions (`getChildContext` etc) in the legacy Form as they break a lot of the input decorators and the work to fix them is out of scope and unnecessary as they are soon to be deprecated by us anyway. The new/ experimental Form has the updated context API stuff

Reply:
Ah OK, thanks for clarifying about that :thumbsup: